lib/error-matcher.js

```diff
@@ -23,12 +23,21 @@ module.exports = (function () {
   util.inherits(ErrorMatcher, EventEmitter);
 
   ErrorMatcher.prototype._goto = function () {
-    var file = this.cwd + path.sep + this.currentMatch[0].file;
-    if (!this.currentMatch[0].file) {
+    var file = this.currentMatch[0].file;
+    if (file) {
+      var oldDir = process.cwd();
+      process.chdir(this.cwd);
```

Comment:
I guess so, the only other way I can think of is re-implement the realpathSync.

Is changing the process cwd actually a bad idea? Is Atom multi-threaded?